.gitignore

```diff
@@ -0,0 +1,21 @@
+# Eclipse
+# -------
+*.project
+*.settings
+
+# IDEA
+# ----
+.idea
+.shelf
+/*.iml
+/*.ipr
+/*.iws
+
+# NetBeans
+# --------
+.nb-gradle
+.nb-gradle-properties
```

Comment:
Nothing Gradle in this repo (nor is it excluding actual Gradle build dirs).

Reply:
It was c/p from another source, I've never used NetBeans, and didn't read the content.

Will trim.